src/TesterInternal/GeoClusterTests/MultiClusterNetworkTests.cs

```diff
@@ -0,0 +1,253 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+using Microsoft.VisualStudio.TestTools.UnitTesting;
+using Orleans;
+using Orleans.Runtime;
+using Orleans.Runtime.Configuration;
+using Orleans.MultiCluster;
+using Orleans.Runtime.MultiClusterNetwork;
+
+namespace Tests.GeoClusterTests
+{
+    // We need use ClientWrapper to load a client object in a new app domain. 
+    // This allows us to create multiple clients that are connected to different silos.
+
+    [TestClass]
+    [DeploymentItem("OrleansAzureUtils.dll")]
+    [DeploymentItem("TestGrainInterfaces.dll")]
+    [DeploymentItem("TestGrains.dll")]
+    [DeploymentItem("ClientConfigurationForTesting.xml")]
```

Comment:
Tests were only partially migrated. That's done now, I have merged latest master.